pages/game_page.py

```diff
@@ -1,3 +1,33 @@
 import streamlit as st
+import random 
 
-st.title("GAME PAGE")
\ No newline at end of file
+st.title("Build a Scientist")
+
+game_words = ['Genetic','Histology','Sequence','Clinical Trial','Mutation','Genome','Oncology']
+
+def game_word_generator(word): 
+    st.write(word)
+    st.write(f"`{' '.join(['_'] * len(word))}`")
+    
+if game_words:
+    selected_word = random.choice(game_words)
+
+game_word_generator(st.session_state.selected_word)
+   
+# Session state
+if 'selected_word' not in st.session_state:
+    st.session_state.selected_word = st.session_state.selected_word
```

Comment:
I would also move this above the call to game_word_generator so that we save selected_word in session_state before we use it
```suggestion
    st.session_state.selected_word = selected_word
```